src/favourites/saga.ts

```diff
@@ -230,6 +233,34 @@ function* removeFavouriteTargetSaga({
   }
 }
 
+function* clearFavouriteSaga(): Generator<Effect, void, never> {
+  try {
+    const oldFavourite: Favourite = yield select(getFavourite);
+
+    if (oldFavourite?.id) {
+      yield call(deleteFavouriteRequest, oldFavourite.id);
+    }
+
+    localStorage.setItem(MVJ_FAVOURITE, JSON.stringify(DEFAULT_FAVOURITE));
```

Comment:
I'm not sure if this is needed. In index.tsx there is this store.subscribe(() => {...}) -function, which updates the localStorage everytime there is something changed in the store. So if you remove the favourite from store, it should be updated in the localstorage also.